package.json

```diff
@@ -20,19 +20,18 @@
     "redux": "^4.0.1",
     "redux-thunk": "^2.3.0",
     "socket.io": "^2.2.0",
-    "socket.io-client": "^2.2.0",
     "socket.io-mock": "^1.2.3"
   },
   "scripts": {
     "start": "node server.js",
-    "start:devserver": "npx cross-env NODE_ENV=testing nodemon server.js --watch ./server --watch ./server.js",
+    "start:devserver": "concurrently \"nodemon ./server/proxy/server.js\" \"npx cross-env NODE_ENV=testing nodemon --inspect server.js --watch ./server --watch ./server.js\"",
```

Comment:
What is this new script doing, you're running two servers? By the name it has something to do with proxying?

Reply:
Yeah, it's supposed to simulate a second client and makes requests to the original server when prompted to from the first client haha, it's a bit confusing, I'll explain at the stand up

Reply:
OK, that's fine 😃

Reply:
cypress is stuck inside a web browser, so all requests it makes will use the same cookie store. this means cypress cannot send requests on behalf of multiple users.  
cypress needs a way to escape the web browser, and control an HTTP client that has a _separate_ cookie store.  
the only way to escape the web platform is via HTTP (we can't run a shell script).  
so, we arrange for a server to be running on 8081. it's a slave that will run any HTTP request you ask.  
and since it's a server (listens for requests from cypress) and also a client (sends HTTP requests to our backend): we call it a proxy. it proxies requests from cypress to the backend.